docs/sources/alerting/configure-notifications/template-notifications/examples.md

```diff
@@ -408,6 +413,73 @@ Execute the template by passing the dot (`.`):
 [FIRING:1, RESOLVED:1] api warning (sql_db)
 ```
 
+## Print a link to a dashboard with time range
+
+You can include a link to a dashboard or panel in your alert notifications. This is useful when the alert rule is created from a dashboard panel or monitors a target visualized in an existing dashboard.
+
+Including a dashboard link in the notification helps responders quickly navigate to the relevant context for investigation.
+
+Use one of the following methods to include a dashboard link with the correct time range in the alert notification:
+
+1. You can [link the alert rule to a panel](ref:link-alert-rules-to-panels). This includes the dashboard and panel URLs via `{{.Alert.DashboardURL}}` and `{{.Alert.PanelURL}}`.
+
+   ```go
+   {{ define "custom.link_to_dashboard" -}}
+   {{ range .Alerts -}}
+     Dashboard: {{.DashboardURL}}
+     Panel: {{ .PanelURL }}
+   {{ end -}}
+   {{ end -}}
+   ```
+
+   Run the template using:
+
+   ```go
+   {{ template "custom.link_to_dashboard" . }}
+   ```
+
+   ```template_output
+   Dashboard: https://example.com/d/uiyahbsdaubsd?from=1740070380000&orgId=1&to=1740074106395
+   Panel: https://example.com/d/uiyahbsdaubsd?from=1740070380000&orgId=1&to=1740074106395&viewPanel=31
+   ```
+
+   These URLs include a time range based on the alert’s timing:
+
+   - `from`: One hour before the alert started.
+   - `to`: The current time if the alert is firing, or the alert’s end time if resolved.
+
+1. Alternatively, you can use a custom annotation to set the dashboard URL and build the full URL using the `from` and `to` query parameters derived from `{{.Alert.StartsAt}}` and `{{.Alert.EndsAt}}`.
+
+   ```go
+   {{ define "custom.my_dashboard_url_annotation" -}}
+   {{ range .Alerts -}}
+
+     {{/* StartsAt - 1h */}}
+     {{ $from := (.StartsAt.Add -3600000000000).UnixMilli }}
+
+     {{ $to := "" }}
+     {{ if eq .Status "resolved" }}
+        {{ $to = (.EndsAt).UnixMilli }}
+     {{ else }}
+       {{/* Use current time if alert is firing */}}
+       {{ $to = (time.Now).UnixMilli }}
+     {{ end }}
+
+     Dashboard: {{.Annotations.MyDashboardURL}}?from={{$from}}&to={{$to}}
```

Comment:
> using a custom annotation and setting up the from and to query params

So, in order for `Dashboard: {{.Annotations.MyDashboardURL}}?from={{$from}}&to={{$to}}` to work, we should add a note to remind users that the alert definition must include the `MyDashboardURL` annotation with a proper base URL. 

Eg.
```
MyDashboardURL: "https://example.com/d/uiyahbsdaubsd"

```

 will correctly render like: Dashboard: https://example.com/d/uiyahbsdaubsd?from=1748842505718&to=1748860505718